### PR TITLE
fix(text-editor): ensure editor resizes with its container

### PR DESCRIPTION
**Pull request by Kiarokh**
and never grows taller or wider than its own container
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/text-editor/text-editor.tsx [hunk 1]
```diff
@@ -82,12 +82,14 @@ export class TextEditor implements FormComponent<string> {
...
-            <fieldset disabled={this.readonly || this.disabled}>
+        return [
+            <span class="notched-outline">
+                <span class="leading-outline" />
                 {this.renderLabel()}
-                {this.renderEditor()}
-            </fieldset>
-        );
+                <span class="trailing-outline" />
+            </span>,
```
Comment (Kiarokh) [inline src/components/text-editor/text-editor.tsx:90]: if you wonder why this refactor was needed, you can see this https://github.com/Lundalogik/lime-elements/pull/2935#discussion_r1581000378

**Comment by civing** [comment]
I get two scrollbars in that example you added 😕

https://github.com/Lundalogik/lime-elements/assets/4279390/a7ac0841-ed1c-4521-b4ea-38c072564b5b

**Comment by Kiarokh** [comment]
> I get two scrollbars in that example you added 😕

hmmm...... good catch.... freaking bloody divs. I'm gonna check this again. Everything was working fine 😢 

Also this has to be fixed:
image

**Review by civing — APPROVED** [review]

**Pull request merged**